### PR TITLE
[Feature] Update SDK to DEV and PRD pipelines and fixed editor

### PR DESCRIPTION
**Pull request by brapoprod**
This PR updates pipelines to go to correct PRD and DEV environments as well as fixing the editor version in place

## Related tickets

-   [WRS-800](https://support.chili-publish.com/projects/WRS/issues/WRS-800)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -70,6 +70,8 @@ export { ColorType } from '../types/ColorStyleTypes';
 
 let connection: Connection;
 
+const FIXED_EDITOR_LINK = 'https://stgrafxstudioprdpublic.blob.core.windows.net/editor/0.0.6/web';
```
Comment (pietervp) [inline src/index.ts:73]: We want to use the CDN link: https://studio-cdn.chiligrafx.com/editor/0.0.6/web

File: .github/workflows/publish-release-npm.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ jobs:
               uses: lauchacarro/Azure-Storage-Action@v1.1
               with:
                   # Connection String of the Azure Storage Container
-                  connection-string: ${{ secrets.AZURE_CDN_SECRET }}
+                  connection-string: ${{ secrets.AZURE_CDN_STUDIO_PRD_CONNECTION_STRING }}
                   # Folder what contains the files for upload to Storage Container
                   folder: upload
```
Comment (pietervp) [inline .github/workflows/publish-release-npm.yml:59]: Not sure what this means for the eventual CDN path of the SDK. It should be `<cdn-domain>/sdk/<version>/abc.js` by the looks of it, with current configuration `/builds/` will be part of the path?

Comment (pietervp) [reply .github/workflows/publish-release-npm.yml:59]: Also, `blob-container-name` should it be `sdk`? It refers to the container on the root level of the Azure Storage account IIRC

File: .github/workflows/publish-package.yml [hunk 3]
```diff
@@ -66,7 +66,7 @@ jobs:
               uses: lauchacarro/Azure-Storage-Action@v1.1
               with:
                   # Connection String of the Azure Storage Container
-                  connection-string: ${{ secrets.AZURE_CDN_SECRET }}
+                  connection-string: ${{ secrets.AZURE_CDN_STUDIO_DEV_CONNECTION_STRING }}
                   # Folder what contains the files for upload to Storage Container
                   folder: upload
```
Comment (pietervp) [inline .github/workflows/publish-package.yml:71]: Probably same issue as the `publish-release-npm.yml`? We want the CDN URL's to be as short as possible

**Review by pietervp — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-release-npm.yml (hunk 2 above)
Comment (brapoprod) [reply .github/workflows/publish-release-npm.yml:59]: blob-container-name should be correct being SDK, storage account = stgrafxstudioprdpublic => container = SDK.
SA is defined by the connectionstring secret

**Review by pietervp — APPROVED** [review]

**Review by alexandraFlavia9 — APPROVED** [review]

**Pull request merged**